src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx

```diff
@@ -146,21 +146,23 @@ You can manage groups via either the UI or via API:
 
-To add access to groups in the UI:
+To create access grants from the UI:
 
-1. From the **Access management** UI, click **Groups**. 
-2. You can create a new group or edit an existing one: 
-  * To create a new group, click **Create new group** and follow the instructions. 
-  * To edit an existing one, click the `...` settings icon for the group you want to edit. Click **View group details** and edit the group's access. 
-
-<img title="New Relic user mgmt - create group" alt="New Relic user mgmt - create group" src={createGroupUi}/>
-<figcaption>On the **Groups** tab, you can click **Create new group** to add a user group.</figcaption>
-
-When you're done, if users are already assigned to the group you created or edited, those users will have the new access within a few minutes (although for [EU region organizations](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), this can take up to twenty minutes or so). If your users are not yet in that group (which would be true if you just created a new group), the next step will show you how to add users. 
+1. From the **Access management** UI, click **Access grants and groups**. 
+2. Choose one of the following:
+* **Existing**: If there is already a group you want to add an access grant to, you can use this. For example, if you want to gives users in the default [**Admin** or **User** group](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#groups) access to new accounts, you might choose this and then select the **Admin Default** or **Admin User** role.
+* **New**: If you need to create a new group, choose this. (This is not available for SCIM provisioning because your groups are managed via SCIM.) 
+3. Next, under **Access grant**, you'll choose one of following:
+* **Account**: Choose this to be able to select from [the roles that are account-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that have to do with using and configuring our observability features (and not about organization and user management).
+* **Organization**: Choose this to be able to select from [the roles that are organization-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that govern organization- and user management. (Note that these users must also already belong to an account-scoped role. This is true for most users but if it's not, you may see a message that the user doesn't belong to an organization.)
```

Comment:
@zuluecho9 do you need the hyphen following "organization"? This doesn't seem like a muti-word adjective.